src/d3a/d3a_core/sim_results/device_statistics.py

```diff
@@ -59,7 +60,7 @@ def _device_price_stats(cls, area: Area, subdict: Dict):
         trade_price_list = []
         for t in market.trades:
             if t.seller == area.name or t.buyer == area.name:
-                trade_price_list.append(t.offer.price / 100.0 / t.offer.energy)
+                trade_price_list.append(t.offer.energy_rate)
```

Comment:
Energy rate that the `trade_price_list` expects is in euros/kWh, while energy rate of the offer is in cents/kWh. I think the /100.0 is still required here.